API.md

```diff
@@ -2,7 +2,7 @@
 # RPC WebSocket API
```

Comment:
`/holo/agent/identify` has been removed from the api so should be removed from this doc.